packages/host/tests/integration/components/prerendered-card-search-test.gts

```diff
@@ -0,0 +1,335 @@
+              module: `${testRealmURL}book`,
+              name: 'Book',
+            },
+          },
+        },
+      },
+      'catalog-entry-1.json': {
+        data: {
+          type: 'card',
+          attributes: {
+            title: 'Post',
+            description: 'A card that represents a blog post',
+            isField: false,
+            ref: {
+              module: `${testRealmURL}post`,
+              name: 'Post',
+            },
+          },
+          meta: {
+            adoptsFrom: {
+              module: `${baseRealm.url}catalog-entry`,
+              name: 'CatalogEntry',
+            },
+          },
+        },
+      },
+      'catalog-entry-2.json': {
+        data: {
+          type: 'card',
+          attributes: {
+            title: 'Article',
+            description: 'A card that represents an online article ',
+            isField: false,
+            ref: {
+              module: `${testRealmURL}article`,
+              name: 'Article',
+            },
+          },
+          meta: {
+            adoptsFrom: {
+              module: `${baseRealm.url}catalog-entry`,
+              name: 'CatalogEntry',
+            },
+          },
+        },
+      },
+    };
+
+    await setupIntegrationTestRealm({
+      loader,
+      contents: {
+        'article.gts': { Article },
+        'blog-post.gts': { BlogPost },
+        'book.gts': BookGtsImpl,
+        'person.gts': { PersonField },
+        'post.gts': { Post },
+        ...sampleCards,
+      },
+    });
+  });
+
+  test(`can search for cards by using the 'eq' filter`, async function (assert) {
+    let query: Query = {
+      filter: {
+        on: {
+          module: `${testRealmURL}book`,
+          name: 'Book',
+        },
+        eq: {
+          'author.firstName': 'Cardy',
+        },
+      },
+      sort: [
+        {
+          by: 'author.lastName',
+          on: { module: `${testRealmURL}book`, name: 'Book' },
+        },
+      ],
+    };
+    let realms = [testRealmURL];
+
+    await render(<template>
+      <PrerenderedCardSearch
+        @query={{query}}
+        @format='embedded'
+        @realms={{realms}}
+      >
+        <:loading>
+          Loading...
+        </:loading>
+        <:response as |cards|>
+          {{#each cards as |card|}}
+            <div class='card-container'>
+              <card.component />
+            </div>
+          {{/each}}
+        </:response>
+      </PrerenderedCardSearch>
+    </template>);
+    await waitFor('.card-container');
+    assert.dom('.card-container').exists({ count: 2 });
+    assert
+      .dom('.card-container:nth-child(1)')
+      .containsText('Card 2 by Cardy Jones');
+    assert
+      .dom('.card-container:nth-child(2)')
+      .containsText('Cardy Stackington Jr. III');
+    assert
+      .dom('.card-container .book')
+      .hasStyle({ backgroundColor: 'rgb(255, 255, 0)' });
+    assert.dom('.card-container .author').hasStyle({ color: 'rgb(0, 0, 255)' });
```

Comment:
By deleting the other test we lost a test that that asserts that style tags do not get inserted twice when multiple prerendered instances of the sam card type are rendered. I thought that was important to have tested however if `loader.import` comes with strong assurances it will not evaluate twice for the same url then I think it's fine to not have that.

Reply:
If there is a way to add this coverage to the pre-rendered-card-search test, we should do so, otherwise it is the cost of keeping the prerendered-card component private, which was important to @ef4